components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIManager.java

```diff
@@ -877,24 +899,25 @@ String addProductResourceFile(APIProductIdentifier identifier, String resourcePa
     
     /**
      * @param searchQuery search query. ex : provider:admin
-     * @param tenantDomain tenant domain
+     * @param orgId Identifier of an organization
      * @param start starting number
      * @param end ending number
      * @return
      * @throws APIManagementException
      */
-    Map<String, Object> searchPaginatedAPIs(String searchQuery, String tenantDomain, int start, int end)
+    Map<String, Object> searchPaginatedAPIs(String searchQuery, String orgId, int start,
+                                            int end)
```

Comment:
not required to change this line?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9609/commits/20a6fb69a612c320d26f57d1d19519722fe30ab5